compute_range.c

```diff
@@ -0,0 +1,91 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
+#include "compute_range.h"
+
+int main()
+{
+  int range_reading[10] = {0,1,2,3,0,6,7,0,10,12};
+  compute_range(range_reading, (sizeof(range_reading)/sizeof(int)));
+}
+
+int cmpfunc (const void * a, const void * b) 
```

Comment:
Call the function by its purpose, not by its code. The code is that it compares. The purpose is to sort ascending.
Try a name like `compareForAscending`

Reply:
updated with new purposeful name.